releases/release-1.32/release-notes/maps/pr-128503-map.yaml

```diff
@@ -0,0 +1,19 @@
+pr: 128503
+releasenote:
+  text: With the CBORServingAndStorage feature gate enabled, built-in APIs can now
```

Comment:
```suggestion
  text: enabled `CBORServingAndStorage feature gate enabled, built-in APIs can now
```

Reply:
Hm, the suggestion reads a little off. I think the original form flows better.

Reply:
How about now ?

Reply:
Better! I think I'll swap the "," for a "-" and that should be good. Thanks for you help :)